### ISSUE TITLE
Consider via_ferrata for hike mode

### ISSUE THREAD

**Issue by karussell** [graphhopper/graphhopper#1326]
See [this route](https://graphhopper.com/maps/?point=46.45199%2C11.65928&point=46.451859%2C11.659058&locale=en&vehicle=hike&layer=TF%20Outdoors). Or is this too dangerous or with [<1000 occasions](https://taginfo.openstreetmap.org/tags/highway=via_ferrata) not worth?

**Comment by hungerburg** [comment]
There is a nice mountain hiking path to Bettelwurfhütte from Absam - https://graphhopper.com/maps/?point=47.307968%2C11.519214&point=47.331019%2C11.510749&point=47.338586%2C11.507562&profile=hike&layer=OpenStreetMap

Remove the via node, graphhopper selects the shorter path over a via ferrata, in both the foot and hike profiles. Is this deliberate?

**Comment by easbar** [comment]
No, GraphHopper should not use the via-ferrata here, especially not for the foot profile.

This is the via-ferrata: https://www.openstreetmap.org/way/128322791
```
access:conditional=no @ (Nov 1-Apr 30)
foot=yes
highway=via_ferrata
name=Absamer Klettersteig (KS4-B bzw. C/D)
source=survey
trail_visibility=good
via_ferrata_scale=3
wikidata=Q30350091
wikipedia=de:Absamer Klettersteig
```

There are multiple reasons GraphHopper still uses it currently:

1) For access tags the most specific tag counts. We have:
```
access:conditional=no @ (Nov 1-Apr 30)
foot=yes
```
So `foot=yes` overrules the (conditional) `access=no` tag. The correct tagging would probably be:
```
access=no
foot:conditional=yes @ (May 1-Oct 31)
```
which means that access is forbidden for all transportation modes, except for pedestrians during summer.

2) GraphHopper normally would not consider `highway=via_ferrata` (hence this issue), but it still does, because of the explicit `foot=yes` tag. Since the current idea is to exclude via-ferratas this is a bug in GraphHopper. 

3) GraphHopper not only sets the access flag for the via-ferrata to 'true', but sets the speed to 5km/h, which would probably be way too fast, even if the via-ferrata was allowed. For this reason it even prefers the via-ferrata over the other route (the one you showed with the via-node).

4) GraphHopper currently does not consider the `via_ferrata_scale`. It does consider the `sac_scale`, though. If the via-ferrata was tagged as `sac_scale=mountain_hiking` or harder (T2 or harder) GraphHopper would at least reduce the speed to 2km/h and it would more likely choose the other route.

Can you comment on the tagging here?

**Comment by hungerburg** [comment]
I never went there, not much of a via_ferrata fan. I stumbled over this just by accidence. From the literature/pictures, this one can be done without the special gear, given limited climbing experience and a bucket head for heights.

I guess, the sign reads "Closed from … to …", and therefore the conditional. Personally, I'd not have tagged foot=yes, it should be taken for granted.

I understand, that tagging there is actually wrong: The more specific rule rightfully overrides the general conditional. Removing foot=yes would be the remedy of my choice. Then tagging should be accurate?

**Comment by easbar** [comment]
> I guess, the sign reads "Closed from … to …", and therefore the conditional.

Yes, then I guess it should be tagged as `foot=yes; foot:conditional=no @ (Nov 1-Apr 30)`. But this won't fix the problem yet, because GraphHopper won't evaluate the conditional tag when there is `foot=yes` also, but this is a bug in GraphHopper IMO. @karussell?

There are quite a few other examples of via-ferratas tagged as `foot=yes` without any conditional access restrictions like this one for example: https://www.openstreetmap.org/way/129725314
In this case `foot=yes` doesn't really do anything, because this is the default anyway. But it isn't wrong to tag it, either. So the bigger problem seems that GraphHopper allows using ways unconditionally when they are tagged as `foot=yes`, even when they are via-ferratas like here.

It looks like you removed the `sac_scale` tag here: https://www.openstreetmap.org/way/128322791 (version 14). Can you explain why? What is the difference between via_ferrata_scale and sac_scale?

**Comment by easbar** [comment]
> Der Vorschlag mit foot:conditional ist besser. Der nächste hängt wieder ein foot=yes dran, wenn ichs einfach nur wegnimm.

Yes. Also because tagging `foot=yes` is not wrong. It is just unnecessary and GraphHopper handles it incorrectly in this case. 

From the [wiki](https://wiki.openstreetmap.org/wiki/Tag:highway%3Dvia_ferrata):

> As used for other ways and relations, [access:conditional](https://wiki.openstreetmap.org/wiki/Key:access:conditional)=* in conjunction with the [opening_hours](https://wiki.openstreetmap.org/wiki/Key:opening_hours)=* syntax can be used to map such restrictions.

This is a bit misleading, because a) opening_hours seems to be less fitting than conditional access(?) and b) the problem we saw here: `access:conditional` could be overruled by the (apparently quite popular) foot=yes tag for these ways. Since there is no transportation mode other than `foot` that can be used on via-ferratas the example should maybe better mention `foot:conditional` instead of `access:conditional`.

Thanks for you explanations regarding sac_scale vs. via_ferrata scale. GraphHopper should either pay attention to `via_ferrata_scale` then or explicitly close all via_ferratas (even those tagged as `foot=yes`).

And maybe `via_ferrata` should be mentioned by the [highway wiki](https://wiki.openstreetmap.org/wiki/Key:highway)

**Comment by hungerburg** [comment]
Die Fehlstellen im Wiki aufgefüllt, nach bestem Wissen. Mal sehen, wie das aufgenommen wird. Der Absamer Klettersteig ist nun nicht mehr missverständlich getaggt. Wenn ich in ein paar Tagen die Route oben aufschlagen werde, dann wird graphhopper nicht mehr dort routen, jetzt im Winter? Aber im Sommer dann wieder?

Sorry, schon wieder auf Deutsch!

I'd say, if a "highway" is based on via_ferrata and not on path, it is not suitable for walking nor for hiking directions. There may be paths with a high via_ferrata_scale though, as some contributors obviously despise the tag. A via_ferrata scale of C likely goes beyond what sac_scale can cover. I do not think any via_ferrata_scale values can be combined with an sac_scale value below alpine_hiking (T4).

**Comment by hungerburg** [comment]
> But why does it exist at all?

I'd say, so that routers have an easy way to tell, where not to send unsuspecting pedestrians, cyclists or horseback riders or whatever transport highway=path is for :) Other reason: There is no path there, just a steel cable :) And occupying highway key enforces that. But this does not square with some mappers desire to map a path everywhere they went or climbed, so it is disputed…

When following the procedural safety measures, i.e. harness with dual carabiner clips and click click every few meters, e.g. the via ferrata mentioned above https://klettersteig.de/klettersteig/absamer_klettersteig/1799 gets 4½ hours ETA from entry point to hut according to guidepost. It may be shorter by distance, even when accounting for the vertical sections, but the number of people that are sooner at the hut than when taking the hiking path, or the number of those, that have to turn around at the first "key-section"?
